keynav.c

```diff
@@ -1440,13 +1536,16 @@ void viewport_left() {
 void handle_keypress(XKeyEvent *e) {
   int i;
   /* If a mouse button is pressed (like, when we're dragging),
-   * then the 'mods' will include values like Button1Mask. 
+   * then the 'mods' will include values like Button1Mask.
    * Let's remove those, as they cause breakage */
   e->state &= ~(Button1Mask | Button2Mask | Button3Mask | Button4Mask | Button5Mask);
 
   /* Ignore LockMask (Numlock, etc) and Mod2Mask (shift, etc) */
   e->state &= ~(LockMask | Mod2Mask);
 
+  /* Ignore different keyboard layouts (e.g. russian) */
+  e->state &= ~(1<<13);
```

Comment:
Is there a constant definition for this?

Reply:
Good point! 
 I think it makes more sense to have a whitelist of modifiers rather than a blacklist, made that change here: https://github.com/mgsloan/keynav/commit/2128fd26b0901f0208cbffdd50bdd6a41495b290